### DISCUSSION TITLE
Possible to do an or if statement in visual builder?

### DISCUSSION THREAD

**Discussion by cstan22** [gopro/labs#1051] [General]
Eg, If current time is less than X or greater than Y, then...

Rather than if current time is less than x, then; if current time is greater than Y, then...

**Comment by dnewman-gpsw** [comment]
Not with OR. So do in in reverse IF(time>X AND time<Y) set variable.

Like this:
![image](https://github.com/user-attachments/assets/ede7d769-f4e7-47de-8b5e-edd7c7bc32c5)

**Reply by cstan22** [reply]
Not sure how to mark something as solved, but this seems like it's it
